index.d.ts

```diff
@@ -1,2 +1,2 @@
-declare function isEqual<A = any, B = any>(a: A, b: B): boolean;
+declare function isEqual<B = any>(a: any, b: B): a is B;
```

Comment:
I don't think there's any need for `= any` now, so I'm going to remove it.
```suggestion
declare function isEqual<B>(a: any, b: B): a is B;
```